src/components/Application/ApplicationStart.tsx

```diff
@@ -10,27 +11,62 @@ export interface ApplicationStartProps {
 }
 
 const ApplicationStart: React.FC<ApplicationStartProps> = (props) => {
-  const { template: type, sections, handleClick } = props
+  const { template, sections, handleClick } = props
+  const { name, code, startTitle, startMessage } = template
 
-  return type ? (
-    <Container text>
-      <Header as="h1" content={type ? type.description : 'Create application page'} />
-      {type && (
-        <Segment>
-          {sections && (
-            <Header as="h2" content={`Number of sections in template: ${sections.length}`} />
-          )}
-          <List>
-            {sections &&
-              sections.map((section) => (
-                <List.Item key={`list-item-${section.code}`} content={section.title} />
-              ))}
-          </List>
-          <Button content={type.name} onClick={handleClick} />
-        </Segment>
-      )}
-      {!type && <Label content="No Application" />}
-    </Container>
+  return template ? (
+    <Segment.Group style={{ backgroundColor: 'Gainsboro', display: 'flex' }}>
+      <Button as="href" icon="angle left" label={{ content: name, color: 'grey' }} />
```

Comment:
Just noticed this also: the `as="href"` is causing this error:
<img width="1140" alt="Screen Shot 2021-01-06 at 10 26 15 AM" src="https://user-images.githubusercontent.com/5456533/103700618-9d5d5780-5009-11eb-845a-672272a3e04d.png">

I'm not sure what this prop is doing -- seems like you can just remove it?

Reply:
It's fixed in the next PR: https://github.com/openmsupply/application-manager-web-app/pull/234/files?file-filters%5B%5D=.png&file-filters%5B%5D=.ts&file-filters%5B%5D=.tsx#diff-4d0ea65adc1a078ca90ef8e261c5544189bd5d38de5c6e051f09f963a6cce354